backends/cuda/ceed-cuda-qfunction.c

```diff
@@ -41,7 +41,7 @@ static int CeedQFunctionApply_Cuda(CeedQFunction qf, CeedInt Q,
   CeedInt numinputfields, numoutputfields;
   ierr = CeedQFunctionGetNumArgs(qf, &numinputfields, &numoutputfields);
   CeedChkBackend(ierr);
-  const int blocksize = ceed_Cuda->optblocksize;
+  const int blocksize = ceed_Cuda->optblocksize / 2;
```

Comment:
Dirty hack to make this work
```console
$ nsys profile --trace=cuda build/solids-elasticity -E 10 -nu .3 -degree 2 -mesh ~/meshes/holes.msh -problem FSInitial-NH1 -num_steps 3 -bc_clamp 1 -bc_traction 2 -bc_traction_2 0,1,0 -view_soln -snes_monitor -ceed /gpu/cuda/shared
```

The default is to use the max number of threads per block (1024), but this requires that register count stays quite low. Real qfunctions can get bigger and we need to choose correspondingly smaller block sizes.